PostSharp.Community.StructuralEquality.Weaver/Subroutines/IgnoredFields.cs

```diff
@@ -28,10 +28,24 @@ public static class IgnoredFields
                 }
                 else if (targetElement is PropertyDeclaration propertyDeclaration)
                 {
-                    FieldDefDeclaration backingField = compilerAdapterService.GetBackingField(propertyDeclaration);
-                    if (backingField != null)
+                    while ( true )
                     {
-                        fields.Add(backingField);
+                        FieldDefDeclaration backingField = compilerAdapterService.GetBackingField(propertyDeclaration);
+                        if ( backingField != null )
+                        {
+                            fields.Add( backingField );
+                        }
+
+                        if ( !propertyDeclaration.Getter.IsVirtual )
+                        {
+                            break;
+                        }
+                        
+                        propertyDeclaration = propertyDeclaration.Parent.BaseTypeDef.FindProperty( propertyDeclaration.Name )?.Property;
+                        if ( propertyDeclaration == null )
+                        {
+                            break;
+                        }
```

Comment:
I don't understand why looking for fields to ignore in base class properties is what we want to do.

Reply:
Removing this feature.